index.js

```diff
@@ -1362,7 +1362,29 @@
             };
           };
         };
-        return S.encase3(f, x, y, z);
+        return encase3(f, x, y, z);
+      });
+
+  //# maybeToEither :: a -> Maybe b -> Either a b
+  //.
+  //. Takes a value of any type and a Maybe, and returns an Either.
+  //. If the second argument is a Nothing, a Left containing the first
```

Comment:
"The Maybe" is *slightly* less precise than "the second argument", since the first argument could also be a Maybe, but I'm happy to make your suggested change.